components/match2/commons/match_group_input.lua

```diff
@@ -576,6 +576,13 @@ function MatchGroupInput.readOpponent(match, opponentIndex, options)
 	end
 
 	Opponent.resolve(opponent, resolveDate, {syncPlayer = true})
+
+	if options.pagifyPlayerNames and opponent.type ~= Opponent.team then
```

Comment:
Why not do it on team oppos?

Reply:
because for team oppos the table doesn't exist yet at that point
while for players k need them with underscores before building name

Reply:
will simplify